server.js

```diff
@@ -17,11 +20,57 @@ app.use(require('morgan')('dev'));
 
 // Routes
 app.get('/', function(req, res) {
-  res.send('Hello, backend!');
+  res.render('index')
 });
 
+app.get('/results', (req, res) =>{
+  let search = req.query.search
```

Comment:
This should be req.query.movie because the input tag on your index.ejs page has the name of "movie"